webpack/assets/javascripts/react_app/components/notifications/NotificationDrawer.js

```diff
@@ -24,6 +24,7 @@ class NotificationDrawer extends Component {
 
   onChange(actionType) {
     switch (actionType) {
+      case ACTIONS.MARK_AS_READ:
       case ACTIONS.RECEIVED_NOTIFICATIONS: {
         this.setState({ notifications: NotificationsStore.getNotifications() });
```

Comment:
fall through switch statements should not be allowed. If a single action name can't describe both cases, refactor and extract a method which can be called from each case.